client/js/components/hyperdrive/index.js

```diff
@@ -24,7 +24,7 @@ if (module.parent) {
       if (entry.type === 'directory') {
         send('archive:update', {root: entry.name})
       } else {
-        send('preview:file', {archiveKey: state.archive.key, entry: entry.name}, noop)
+        send('preview:file', {archiveKey: state.archive.key, fileName: entry.name}, noop)
```

Comment:
only comment, otherwise all looks good: maybe we should stick with original nomenclature around archive, so instead of `state.archive.fileName`, we go for `state.archive.entry`. i stubbed it out poorly in the model :)